### DISCUSSION TITLE
Connection Issue with Google Drive - "This app is blocked"

### DISCUSSION THREAD

**Discussion by khanhndevizi** [iterate-ch/cyberduck#16209] [Q&A]
Dear Mountain Duck Community,

I am currently experiencing an issue when trying to connect Mountain Duck to my Google Drive account. Specifically, after following the login steps, I encountered the error message: "This app is blocked."

I have attempted to resolve the issue by checking access permissions and logging in again, but the problem persists. I would greatly appreciate it if your community support team could provide guidance on how to resolve this issue.

Thank you for your assistance, and I look forward to your prompt response.

![This app is blocked](https://github.com/user-attachments/assets/4e4abb07-a3d9-4a7a-88da-63c37cedabd4)

**Comment by dkocher** [comment]
In #16210.
